task10/index.js

```diff
@@ -0,0 +1,63 @@
+const div = document.getElementById("js-parent");
+
+const loading = () => {
+    const loadingPlace = document.createElement("div");
+    const gif = document.createElement('img');
+    loadingPlace.id = "loadingPlace";
+    gif.src = "loading-circle.gif";
+    div.appendChild(loadingPlace).appendChild(gif);
+};
+
+const removeLoading = () => {
+    document.getElementById("loadingPlace").remove();
+};
+
+const getData = () => {
+    const obj = [
+        {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+        {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+    ]
+    
+    return new Promise((resolve,reject) => {
+        setTimeout(() => {
+            resolve(obj);
+            reject("error!");
+        },2000);
+    });    
+};
+
+const objShow = async () => {
+    loading();
+    
+    const value = await getData();
```

Comment:
You can use try-catch-finally here.

Reply:
I asked the question on discord because my interpretation might be wrong. Please confirm.
I am commenting it out for the time being.